include/aspect/material_model/interface.h

```diff
@@ -516,6 +516,39 @@ namespace aspect
     };
 
 
+    /**
+     * A class for additional output fields to be added to the RHS of the
+     * Stokes system, which can be attached to the
+     * MaterialModel::MaterialModelOutputs structure and filled in the
+     * MaterialModel::Interface::evaluate() function.
+     */
+    template<int dim>
+    class AdditionalMaterialOutputsStokesRHS: public AdditionalMaterialOutputs<dim>
+    {
+      public:
+        AdditionalMaterialOutputsStokesRHS(const unsigned int n_points,
+                                           const unsigned int n_stokes_components)
```

Comment:
Normally dim+1 but dim+2 if you have melt transport enabled. That is also the reason I did not want to use
```
Tensor<1,dim> u;
double p;
double p_c;
```
or something of that matter.